bevy_lint/README.md

```diff
@@ -1,9 +1,151 @@
+<div class = "rustdoc-hidden">
+
 # `bevy_lint`
 
 `bevy_lint` is a custom linter for the [Bevy game engine](https://bevyengine.org), similar to [Clippy](https://doc.rust-lang.org/stable/clippy).
 
+</div>
+
+<div class="rustdoc-alert rustdoc-alert-warning">
+
+> **Warning**
+>
+> This is an unofficial community project, hacked upon by the Bevy CLI working group until it is eventually upstreamed into the main [Bevy Engine organization](https://github.com/bevyengine). Pardon our rough edges!
+
+</div>
+
+## Installation
+
+`bevy_lint` depends on a pinned nightly version of Rust with the `rustc-dev` Rustup component. You can refer to the [compatibility table](#compatibility) to see which version of the linter requires which toolchain.
```

Comment:
It would be nice to have a sentence here explaining why this annoying requirement exists.